.github/workflows/compilation.yml

```diff
@@ -198,9 +198,6 @@ jobs:
         automatic_release_tag: "latest"
         title: "${{ steps.version.outputs.version }}"
         files: |
-          OPNPS2LD-${{ env.OPL_VERSION }}.7z
```

Comment:
Id rather keep that for pre-release too. I remember that this was done specially for your scripts that were not able to correctly parse OPNPS2LD-${{ env.OPL_VERSION }}.7z naming

Reply:
Yes, it was intended for direct download links

But duplicating the release assets only confuses people.

And the version number is clearly visible on the heading of the release card

Reply:
But after downloading you will loose that info. If you will have tons of OPNPS2LD.7z archives its a mess to understand which one is correct. I agree that packages should be reduced, but I am against the idea to keep commit-less naming. The only reason to keep short version of archive names, is your scripts, so I advice to change scripts to be able to parse more complex naming.